docs/help/security.mdx

```diff
@@ -0,0 +1,242 @@
+- **JWT Mode:** Integrate with popular authentication providers like Auth0, AWS Cognito, Firebase, and Clerk.
+- **Webhook Mode:** Use a custom webhook for authentication in unique scenarios.
+- **NoAuth Mode:** For testing or development, or for fully public APIs, you can use NoAuth mode, which requires no
+  authentication.
+
+### Multi-Factor Authentication (MFA)
+
+Currently, Hasura DDN does not natively support MFA for console login. However, the recommended approach for enterprise
+clients is to integrate with a corporate Identity Provider (IDP) via SAML, allowing you to leverage your existing
+identity and multi-factor authentication infrastructure, and ensuring a consistent, and secure, access management
+process.
+
+If you use a corporate email address to sign up for Hasura DDN, the password you choose is tied only to our system and
+you can set it to whatever you'd like.
+
+## Self-Hosted Data Plane
+
+For enhanced security and control, Hasura Private DDN allows the data plane to be hosted within your own infrastructure.
+This ensures that the data processing and API execution occur within your controlled environment, providing an added
+layer of security. [Learn more](/private-ddn/overview.mdx).
+
+## Access Tokens
+
+Hasura DDN provides two types of access tokens for supergraph management: Personal Access Tokens (PATs) and Service
+Account Tokens.
+
+### Personal Access Tokens (PATs)
+
+Personal Access Tokens (PATs) are designed for individual users to interact with the Hasura DDN Control Plane. These
+tokens are tied to your user account and are ideal for development, testing, and accessing the Hasura Console.
+
+- **Usage:** Primarily used for accessing the Hasura Console and authenticating the DDN CLI during development and
+  testing.
+- **Security:** Linked to a user account; not to be shared or used in production environments.
+- **Generation:** Create and manage PATs through your
+  [account settings](https://cloud.hasura.io/account-settings/access-tokens) on the Hasura Cloud dashboard or via the
+  DDN CLI with the `ddn auth login` command and `ddn auth print-pat` command.
+
+### Service Account Tokens
+
+Service Account Tokens are designed for applications, services, or automated processes that need to interact with the
+Hasura DDN Control Plane.
+
+- **Usage:** Ideal for continuous integration and continuous deployment (CI/CD) pipelines, automated scripts, and
+  backend services.
+- **Security:** Associated with a project, not an individual user.
+- **Generation:** You can create new service accounts and manage tokens for existing ones via your project's
+  `Settings` > `Service Accounts` tab of the console.
+
+## Securely Managing Secrets with Environment Variables
+
+Hasura DDN strongly recommends using environment variables to manage sensitive information, such as API keys, database
+connection strings, and other credentials. This practice enhances security by keeping secrets out of your codebase and
+metadata files.
+
```

Comment:
Need to add information about how we securely store .env.cloud information on our cloud. (We store it in a vault).